src/zsf_config.h

```diff
@@ -0,0 +1,30 @@
+
+#pragma once
+
+#ifndef _ZSF_CONFIG_H
+#define _ZSF_CONFIG_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "sealock.h"
+
+#  define ZSF_MAX_LOCKS 50
```

Comment:
const int

Reply:
Such a construct is completely not in line with generally prevailing C style guides. Won't fix.

Reply:
I can't find those style guides (kernel style guide only says 'Enums are preferred when defining several related constants' https://www.kernel.org/doc/html/v4.10/process/coding-style.html, but I will trust you if you say that in idiomatic C integer constants are defined by macros.